### PR TITLE
Fixes from the plc bugbash

### PR DESCRIPTION
**Pull request by mehalshah**
-Move the submit button to the right and make it orange with white text (checked with tanya) on the preview assignments page
-Add 'leaving' to the external link description text
-Bullets should not be quoted

### CODE REVIEW

File: dashboard/config/scripts/levels/Wednesday Report: Discovery Learning in Unit 1.level [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
+    "markdown_instructions": "At this point you've seen 7 of the 14 lessons from unit 1. Thinking about those lessons, and looking across the other lessons in <a href=\"curriculum.code.org/csp/unit1/\" target=_blank>unit 1</a>. Again, consider the following questions with your partner:\r\n\r\n- What opportunities have you seen for learners to discover concepts for themselves? \r\n- How did the process of engaging in discovery activities change the pacing, flow, or outcome of the lesson? \r\n- What benefits and pitfalls did you see in discovery activities? \r\n\r\n<br>\r\n\r\nOnce you've had a chance to think about what you've seen today,  work with your partner to develop a set of recommendations about balancing the teacher and the tools in Unit 1. Your recommendations should include:\r\n\r\n- Which lessons or activities provide opportunities for discovery learning? \r\n- What recommendations do you have for using discovery learning in Unit 1?  \r\n\r\n<br><br>\r\nUse the space below to record your recommendations (you'll share them tomorrow). ",
```
Comment (joshlory) [inline dashboard/config/scripts/levels/Wednesday Report: Discovery Learning in Unit 1.level]: This is going to conflict with my content PR.  I've already fixed this level in https://github.com/code-dot-org/code-dot-org/pull/9021.

**Comment by joshlory** [comment]
LGTM after pulling out the change to the `.level` file.

File: dashboard/config/scripts/levels/Wednesday Report: Discovery Learning in Unit 1.level (hunk 1 above)
Comment (mehalshah) [inline dashboard/config/scripts/levels/Wednesday Report: Discovery Learning in Unit 1.level]: Oh - I thought I was fixing that according to the highlights. I'll pull this out.

**Comment by mehalshah** [comment]
Removed the .level file. Waiting for circle now

**Pull request merged**